composer.json

```diff
@@ -3,7 +3,7 @@
     "description": "Programatically convert image formats",
     "type": "silverstripe-vendormodule",
     "require": {
-        "silverstripe/assets": "^1.4 || ^2.0"
```

Comment:
So we should change the existing version to ^1.4 only, and fork this to a new major version for ^2.2 right?

Reply:
I would leave the existing version at 1.4 and 2.0, otherwise you cant install the module for SS 5.0 and 5.1.

The new version needs a new major for 2.2, yes.